sdk/resourcemover/Azure.ResourceManager.ResourceMover/tests/ResourceMoverManagementTestBase.cs

```diff
@@ -30,12 +34,55 @@ public void CreateCommonClient()
             Client = GetArmClient();
         }
 
-        protected async Task<ResourceGroupResource> CreateResourceGroup(SubscriptionResource subscription, string rgNamePrefix, AzureLocation location)
+        protected async Task<ResourceGroupResource> CreateResourceGroup(SubscriptionResource subscription, string rgName, AzureLocation location)
         {
-            string rgName = Recording.GenerateAssetName(rgNamePrefix);
             ResourceGroupData input = new ResourceGroupData(location);
             var lro = await subscription.GetResourceGroups().CreateOrUpdateAsync(WaitUntil.Completed, rgName, input);
             return lro.Value;
         }
+
+        protected async Task<MoverResourceSetResource> CreateMoverResourceSet(ResourceGroupResource rg, string moverResourceSetName)
+        {
+            MoverResourceSetData input = new MoverResourceSetData(AzureLocation.EastUS2)
+            {
+                Properties = new MoverResourceSetProperties(AzureLocation.EastUS, AzureLocation.EastUS2),
+                Identity = new ManagedServiceIdentity(ManagedServiceIdentityType.SystemAssigned)
+            };
+            var lro = await rg.GetMoverResourceSets().CreateOrUpdateAsync(WaitUntil.Completed, moverResourceSetName, input);
+            return lro.Value;
+        }
+
+        protected async Task<VirtualNetworkResource> CreareVirtualNetwork(ResourceGroupResource rg, string vnetName)
+        {
+            VirtualNetworkData virtualNetworkData = new VirtualNetworkData()
+            {
+                Location = AzureLocation.EastUS,
+                Subnets =
+                {
+                    new SubnetData()
+                    {
+                        Name = "Subnet-0824",
```

Comment:
should we use Recording.GenerateAssetName to generate a name for this subnet?

Reply:
Fixed